java/mykola-melnyk/src/com/zoolatech/lecture3/tasks/_4/CsvReporter.java

```diff
@@ -0,0 +1,27 @@
+package com.zoolatech.lecture3.tasks._4;
+
+public class CsvReporter extends Reporter {
+    CsvReporter(String headerData, String fileData, String fileName){
+        super(headerData, fileData, fileName);
+    };
+
+    @Override
+    public void createFile() {
```

Comment:
It seems that methods `createFile` and `closeFile` have a common implementation. I think we can move them to the parent class, so the code won't be duplicated.

Reply:
Done